src/client/app/components/admin/CreateUserModalComponent.tsx

```diff
@@ -0,0 +1,167 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
```

Comment:
I want to understand how this relates to PR #1306 that created cards/modal for users. First, if it is the same basic work then this PR must be held until that work is review/approved in the separate PR. Second, the plan was to remove user from the admin panel (now site settings) so not sure if/where this belongs.